gst/nnstreamer/tensor_query/tensor_query_server.c

```diff
@@ -0,0 +1,47 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * Copyright (C) 2021 Samsung Electronics Co., Ltd.
+ *
+ * @file    tensor_query_server.c
+ * @date    03 Aug 2021
+ * @brief   GStreamer plugin to handle meta_query for server elements
+ * @author  Junhwan Kim <jejudo.kim@samsung.com>
+ * @see     http://github.com/nnstreamer/nnstreamer
+ * @bug     No known bugs
+ *
+ */
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include "tensor_query_server.h"
+#include <nnstreamer_util.h>
+#include <tensor_typedef.h>
+#include <tensor_common.h>
+
+/**
+ * @brief sink config is shared for src
+ */
+static GstTensorsConfig sink_config;
+
+/**
+ * @brief set sink config 
+ */
+void
+gst_tensor_query_server_set_sink_config (GstTensorsConfig * config)
+{
+  sink_config.rate_n = config->rate_n;
+  sink_config.rate_d = config->rate_d;
+  gst_tensors_info_copy (&sink_config.info, &config->info);
```

Comment:
minor things. L42 also.
```suggestion
  gst_tensors_config_copy (&sink_config, &config);
```

Reply:
Updated! Thanks